File: Permutations/permutations.java

import java.util.Scanner;

public class permutations{
    public static void main(String args[]){
        int f1,f2,n,r,num,nper;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter number of seats");
        n = sc.nextInt();
        System.out.println("Enter number of selected seats");
        r = sc.nextInt();
        f1=1;
        for(int i=n;i>1;i--){
            f1*=i;
        }
        num=n-r;
        f2=1;
        for(int j=num;j>1;j--){
            f2*=j;
        }
        nper =f1/f2;
        System.out.println(nper+" ways");
        
    }
}

File: Permutations/permutations.py
n=int(input("Enter number of seats "))
r=int(input("Enter number of people "))
f1=1
f2=1
for i in range(2,n+1):
    f1*=i

num=n-r
for i in range(2,num+1):
    f2*=i
nper=f1//f2
print("Number of ways: ",nper)
